File: week07/mapreduce/hadoop/wordcount_java/src/test/java/wordcount/WordCountTest.java
package wordcount;

import org.apache.hadoop.conf.Configuration;
import org.apache.hadoop.util.ToolRunner;

public class WordCountTest {

  public static void main(String[] not_used) throws Exception {
    String[] args = {"src/test/resources/the_little_prince.txt"};

    Configuration conf = new Configuration();
    conf.setInt("mapreduce.job.reduces", 3);

    ToolRunner.run(conf, new WordCount(), args);
  } // end method main.

} // end class WordCountTest.

File: week07/mapreduce/hadoop/wordcount_scala/src/test/scala/wordcount/WordCountTest.scala
package wordcount

import org.apache.hadoop.conf.Configuration
import org.apache.hadoop.util.ToolRunner

object WordCountTest {

  def main(not_used: Array[String]): Unit = {
    val args = Array("src/test/resources/the_little_prince.txt")

    val conf = new Configuration()
    conf.setInt("mapreduce.job.reduces", 5)

    ToolRunner.run(conf, new WordCount, args)
  } // end method main.

} // end class WordCountTest.
